src/corehost/cli/test/nativehost/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+cmake_minimum_required (VERSION 2.8.12)
+project(nativehost)
+
+set(DOTNET_PROJECT_NAME "nativehost")
+
+set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
```

Comment:
Should this OSX specific stuff be moved to the `testexe.cmake`? So that all our test exes share it. I admit I don't really know what this code does, but if we need it for native host, we might need it for other tests as well.

Reply:
This is setting the search path for the executable to load libraries at runtime. For the native host, I made it look right next to itself (the tests copy nethost next to it). Other executables might not need to add to the search path all or might want to add something different, so I think it makes sense to leave it out of `testexe.cmake` (at least for now - if we do end up with a bunch of test executables that want the same thing, we can move it)